### ISSUE TITLE
No interactive prompt

### ISSUE THREAD

**Issue by techiew** [xhcoding/sshpass-win32#4]
I want to login with SSH using sshpass. It logs in successfully and prints the prompt but the prompt is not interactive and I cannot see anything I write.

**Comment by cxzt** [comment]
me too

**Comment by molocko** [comment]
same

**Comment by robbely** [comment]
Same here, I'm guessing this is no longer maintained.

**Comment by reflection1921** [comment]
I have submitted a PR with a commit that fixes this issue.  (#6) 
Until the PR is reviewed and merged, you can download the source code from my forked repository and build it.
- Repo: [Forked Repo][https://github.com/reflection1921/sshpass-win32]

**Comment by reflection1921** [comment]
> > I have submitted a PR with a commit that fixes this issue. ([#6](https://github.com/xhcoding/sshpass-win32/pull/6)) Until the PR is reviewed and merged, you can download the source code from my forked repository and build it.
> > ```
> > * Repo: [Forked Repo](https://github.com/reflection1921/sshpass-win32)
> > ```
> 
> Many thanks :) If possible, could you release a binary.

I have uploaded the binary to the Releases section of our repository. Please check it out.
- [Releases](https://github.com/xhcoding/sshpass-win32/releases)

**Comment by robbely** [comment]
Many thanks, this has fixed the issue

**Comment by reflection1921** [comment]
That's right. I have modified the link for someone who will see this later. Thank you!

**Comment by xhcoding** [comment]
fixed by https://github.com/xhcoding/sshpass-win32/pull/6
